security/oc_pstat.c

```diff
@@ -483,12 +483,17 @@ oc_pstat_handle_target_mode(size_t device, oc_dpmtype_t *tm)
   if (*tm == OC_DPM_NSA) {
```

Comment:
At line 273 we need to reset sw resource to default values and store it to storage.

Reply:
fixed